integration-tests/src/test/java/org/zowe/apiml/integration/authentication/services/ZaasClientIntegrationTest.java

```diff
@@ -68,7 +68,7 @@ class ZaasClientIntegrationTest implements TestWithStartedInstances {
     ZaasClient tokenService;
 
     private static String getAuthHeader(String userName, String password) {
-        String auth = userName + ":" + password;
+        String auth = userName + ":" + (password == null ? null : password);
```

Comment:
This change does not make any sense.
```suggestion
        String auth = userName + ":" +password;
```